blog/_posts/2018-02-07-collections-performance.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy
+title: On Performance of the New Collections
+---
+
+In a [previous blog post](/blog/2017/11/28/view-based-collections.html), I explained
```

Comment:
I would group all those paragraphs. until `Overhead Of View Based Implementations`